src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -2735,6 +2795,22 @@ public override Breakpoint DisableBreakpoint(Breakpoint breakpoint)
             return null;
         }
 
+        private Debugger GetRunspaceDebugger(int runspaceId)
+        {
+            if(!Runspace.RunspaceDictionary.TryGetValue(runspaceId, out WeakReference<Runspace> wr))
+            {
+                throw new PSArgumentException("Invalid runspace id.");
+            }
+
+
+            if(!wr.TryGetTarget(out Runspace rs))
+            {
+                throw new PSArgumentException("Unable to get Runspace.");
```

Comment:
Seems like these messages could be localized so should be in a resx file

Reply:
done